web_src/less/_base.less

```diff
@@ -1196,11 +1210,7 @@ a.ui.card:hover,
   }
 
   .searchbox {
-    background-color: #f4f4f4 !important;
-
-    &:focus {
-      background-color: #e9e9e9 !important;
-    }
+    background-color: var(--color-input-background) !important;
```

Comment:
Note, this `searchbox` is never actually visible and is TODO in template since 4 years apparently. Still, I fixed its colors 😆